### DISCUSSION TITLE
Amcrest 1277EW vs 1179EW Recommendation

### DISCUSSION THREAD

**Discussion by YenLegion** [blakeblackshear/frigate#7987] [Ask A Question]
I see one of the recommended cameras got a refresh. Wouldn't be using the 'AI' functionality of the 1277 but it does come with color night vision, that's the main difference I am seeing. I am looking to get another camera and have the 1179 but don't know whether to get another one or try the update, it's about the same price. Any insight? Thanks.

**Comment by hawkeye217** [comment]
If you're looking for the best color vision at low light levels without using IR, look into the EmpireTech/Dahua 5442 series. It's a PoE cam (not wifi) and a bit more expensive than the Amcrest, but performance is much, much better due to the sensor size and lens.
